x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/predicate/operator/arithmetic/Neg.java

```diff
@@ -30,12 +32,19 @@
 import static org.elasticsearch.xpack.esql.type.EsqlDataTypes.isTemporalAmount;
 
 public class Neg extends UnaryScalarFunction {
-
-    private final Warnings warnings;
```

Comment:
Was this just unused? (I'd expect it to be - this is the wrong place for a warnings object)

Reply:
Yeah. It's unused. Sorry for not commenting.